ObjectiveGit/GTCredential.h

```diff
@@ -74,6 +74,18 @@ NS_ASSUME_NONNULL_BEGIN
 /// Return a new GTCredential instance, or nil if an error occurred
 + (nullable instancetype)credentialWithUserName:(NSString *)userName publicKeyURL:(nullable NSURL *)publicKeyURL privateKeyURL:(NSURL *)privateKeyURL passphrase:(nullable NSString *)passphrase error:(NSError **)error;
 
+/// Create a credential object from a SSH keyfile data string
+///
+/// userName         - The username to authenticate as. Must not be nil.
```

Comment:
Happily I think we can stop writing `nil`ability in the comments, since `nullable` indicates it!